Source/HarmonyPatches/Patches_Secuirity/Patch_EscortPrisoner.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using HarmonyLib;
+using PrisonLabor.Core.LaborArea;
+using RimWorld;
+using Verse;
+using Verse.AI;
+
+namespace PrisonLabor.HarmonyPatches.Patches_Secuirity
+{
+    [HarmonyPatch(typeof(JobDriver_TakeToBed))]
+    [HarmonyPatch("MakeNewToils")]
+    public static class Patch_EscortPrisoner
```

Comment:
Prisoner would not return to cell by himself? I dont like the idea about wardens handling prisoners all the time. It's not a prison architect.

Reply:
Prisoner do return to cells. I've added a new "JobDriver_FreePrisonerTime" that is last on thinktree so they go back to their cells on their own. Also this patch is so guards stop grabbing prisoner when working outside their cells.